src/bodywork/workflow_execution.py

```diff
@@ -462,7 +455,7 @@ def _remove_readonly(func: Any, path: Any, exc_info: Any) -> None:
         os.chmod(path, stat.S_IWRITE)
         func(path)
     else:
-        raise Exception
+        _log.warning(f"Could not remove file/directory: {path}")
```

Comment:
It's an error, isn't it?

Getting tired now, so I may not be thinking straight.

Reply:
This is just used for cleaning up the cloned repo directory at the end of the workflow therefore if it can't do it for whatever reason it shouldn't result in an otherwise successful workflow, reporting as failed (non zero exit code).

Reply:
Agreed.